statsmodels/tools/tools.py

```diff
@@ -511,3 +513,36 @@ def __init__(self, **kw):
                       message='webuse will be removed from the tools '
                               'namespace in the 0.7.0 release. Please use the'
                               ' new import.')
+
+
+def _ensure_2d(x, ndarray=False):
+    """
+
+    Parameters
+    ----------
+    x : array, Series, DataFrame or None
+        Input to verify dimensions, and to transform as necesary
+    ndarray : bool
+        Flag indicating whether to always return a NumPy array
+
+    Returns
+    -------
+    out : array, DataFrame or None
+        array or DataFrame with 2 dimensiona.  One dimensional arrays are
+        returned as nobs by 1. None is returned if x is None.
+
+    Notes
+    -----
+    Accepts None for simplicity
+    """
+    if x is None:
+        return x
+    if x.ndim == 2:
+        return x
+    elif x.ndim > 2:
+        raise ValueError('x mst be 1 or 2-dimensional.')
+
+    if ndarray or not _is_using_pandas(x, None):
+        return np.asanyarray(x)[:, None]
```

Comment:
use asarray not asanyarray  (we don't want subtypes of array in the calculation)